tests/conftest.py

```diff
@@ -11,11 +11,12 @@
 from sqlalchemy.orm import sessionmaker
 pytest_plugins = "smtpdfix"
 
-SQLALCHEMY_TEST_DATABASE_URL = "sqlite:///./test.db"
+SQLALCHEMY_TEST_DATABASE_URL = "postgresql://postgres:1234@localhost/postgres"
```

Comment:
Could you please explain why?

Reply:
The full-text search feature works on a PSQL environment only.

Reply:
So maybe you could do the same trick as you did for the app environment? :)